dsl/src/commonMain/kotlin/it/unibo/collektive/aggregate/AggregateContext.kt

```diff
@@ -116,6 +117,27 @@ class AggregateContext(
         return body().also { stack.dealign() }
     }
 
+    /**
+     * Projects the field to be aligned with the neighbours.
+     * A field de-alignment can occur when the field is used inside a body of a branch condition.
+     * Take the [field] to project and returns a new field aligned with the neighbours.
+     * This method is meant to be used internally by the compiler plugin.
```

Comment:
```suggestion
     * This method is meant to be used internally by the Collektive compiler plugin, and will solely result into additional and unnecessary messages exchanged if used manually.
```

Reply:
Although, if it's something that we only want to be available in the compiler plugin, it should be moved in the compiler plugin...